expression/builtin_time_vec_test.go

```diff
@@ -87,7 +87,11 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 	ast.CurrentDate: {
 		{retEvalType: types.ETDatetime},
 	},
-	ast.MakeDate: {},
+	ast.MakeDate: {
+		{retEvalType: types.ETDatetime, childrenTypes: []types.EvalType{types.ETInt, types.ETInt},
+			geners: []dataGenerator{&rangeInt64Gener{1700, 2200}, &rangeInt64Gener{0, 365}},
```

Comment:
I'm confused with the low bound of 1700,  IMO, the first argument can be smaller than 1700.

Reply:
Fixed.